### ISSUE TITLE
Feedbacks sur l'abstract et le titre

### ISSUE THREAD

**Issue by hsablonniere** [hsablonniere/talk-rex-components-modules-cdn#1]

**Comment by k33g** [comment]
Mes titres préférés:
- Le monde du JavaScript est devenu fou, un retour à la simplicité est-il encore possible ?
- Dans cette jungle de l’outillage JavaScript, un retour à la simplicité est-il encore possible ?

Abstract:
- 1ère lecture, ça fait envie d'assister à la prez
- Au niveau de `jusqu’à quel point on peut revenir à des bases plus simples en pesant les avantages et inconvénients.` je donnerais un ou deux exemples (minification ou pas, transpilation "on the fly" *je te laisse trouver les ex appropriés*) : ça permet aux lecteurs de se projeter

**Comment by jlengrand** [comment]
Coucou, 

On en a déjà un peu parlé, ton sujet déchire. 

Le monde du JavaScript est devenu un peu fou. Cette jungle d’outils est dense et seuls les plus "veille" peuvent s’y retrouver. Ça n’est pas une situation acceptable et il faut rester vigilants. Entre ceux qui disent que c’était mieux avant et les drogués aux derniers outils à la mode, il faut trouver un juste milieu. -> Je trouve que tu défonces un peu des portes ouvertes, y a tellement de memes sur javascript déjà. Mais je pense que c'est parce que tu es concis dans ta conclu. 

Les take aways sont dingues!

Je ne sais pas si tu l'as vue mais ton plan me fait bcp penser à cette prés : https://youtu.be/kAD7qKwuJFo?t=7928.

Quand je vois la quantité dingue de trucs que tu veux aborder, je me demande si tu vas vraiment faire honneur (avoir le temps de faire piger les gens) à ton outil plus loin qu une démo anecdotique ou d illustration. On a presque deux sujets différents - 1 - l evolution et le pourquoi et le fait que c est possible de faire simple sans trop de problemes - 2 - ton outil qui s appuie sur ça pour faire du catering ultime pour l utilisateur.

Ou alors faire un topo en fin de conf avant / apres. Mais par example dans ta conclu actuelle tu ne mentionnes même pas ton outil, donc on se demande la place qu'il a vraiment

Pareil que pour @k33g pour les titres.


Excellent, sign me up@

**Comment by hsablonniere** [comment]
Merci les gens ;-)

**Comment by hsablonniere** [comment]
@jlengrand Là c'est un document de travail hein, à l'écriture, on verra ce qui rentre mais oui, comme souvent, j'ai du mal à trier et c'est un peu ambitieux. Je crois que j'avais vu le talk de Fred, je ne me souviens plus bien mais je vais me le refaire. Merci pr les feedbacks.

**Comment by jlengrand** [comment]
Yes, j'ai bien compris <3. Mais c'est le seul truc que j'ai en face de moi à review. Bon courage!

**Comment by LostInBrittany** [comment]
Désolé pour le retard...

L'abstract donne très envie, carrément !!!!! Le sujet me parle bien (comme tu sais) et je trouve l'approche super pertinente.

Je coincide avec @jlengrand que ça semble super ambitieux, ça donne pour une université de 3 heures (qui serait géniale, d'ailleurs).  Et la partie la plus compliquée me semble donc voir ce qu'il faudra *edit out*. Mais l'abstract en lui même déchire.

**Comment by k33g** [comment]
@hsablonniere perso je pense que ça rentre (pour t'avoir vu en prez et connaissant ton rythme) en racontant une histoire et en choisissant des exemples percutants mais pas trop nombreux, pas de live coding, plutôt de l'exécution (ou du copier coller de code avant exécution)
